cpp/src/feather/CMakeLists.txt

```diff
@@ -21,6 +21,16 @@ set(LIBFEATHER_SRCS
   status.cc
   types.cc
   writer.cc
+  api.h
+  buffer.h
+  common.h
+  feather-c.h
+  io.h
+  metadata.h
+  reader.h
+  status.h
+  types.h
+  writer.h
```

Comment:
This isn't strictly necessary, it just makes it so the header files are pulled into the MSVC solution, compiles either way.